spec/movements_spec.rb

```diff
@@ -0,0 +1,49 @@
+$LOAD_PATH << File.expand_path('. ./lib', __FILE__)
+# require 'martian_robots'
+require 'movements'
+
+RSpec.describe Movements do
+
+  let (:movements) {Movements.new}
+
+  it "returns E if robot's initial orientation is N" do
```

Comment:
You could wrap your test descriptions in another `describe` or `context` to give more context.
For instance, you could do this:
```
...
describe "turning clockwise" do
  it "turns the robot's orientation from N to E" do
  ...
  end
...
end 
...
```